modules/AsyncProps.js

```diff
@@ -112,7 +118,7 @@ function createElement(Component, props) {
     return <Component {...props}/>
 }
 
-export function loadPropsOnServer({ components, params }, cb) {
+export function loadPropsOnServer({ components, params }, cb, resolver) {
```

Comment:
lets put the resolver in the object with `components` and `params`. Gives nice symmetry between component props and static function "props".